bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -122,6 +122,9 @@ HTTP_PROPERTIES=$(cat <<-EOF
 http.enabled=${KAFKA_BRIDGE_HTTP_ENABLED}
 http.host=${KAFKA_BRIDGE_HTTP_HOST}
 http.port=${KAFKA_BRIDGE_HTTP_PORT}
+http.cors.enabled=${KAFKA_BRIDGE_CORS_ENABLED}
+http.cors.allowedOrigins=${KAFKA_BRIDGE_CORS_ALLOWED_ORIGINS}
+http.cors.allowedMethods=${KAFKA_BRIDGE_CORS_ALLOWED_METHODS}
```

Comment:
@laidan6000 @PaulRMellor where we should document these new coming fields for the bridge standalone installation?

Reply:
@ppatierno thanks Paolo. as discussed, will pick up the doc